.github/workflows/cli-test.yml

```diff
@@ -17,6 +17,8 @@ jobs:
   find-nightly:
```

Comment:
Should we set `permissions: {}` on the top-level so any new jobs that are defined won't get permission by default? Alternatively, we could also set `permissions: { contents: read }` at the workflow-level and keep the `permissions` block on the `report-failure` job.

Reply:
Sure.